src/ICal/ICal.php

```diff
@@ -314,7 +322,7 @@ protected function initLines(array $lines)
             foreach ($lines as $line) {
                 $line = rtrim($line); // Trim trailing whitespace
                 $line = $this->removeUnprintableChars($line);
-                $line = $this->cleanData($line);
+                if (!$this->disableCharacterReplacement) $line = $this->cleanData($line);
                 $add  = $this->keyValueFromString($line);
```

Comment:
Can you remove the extra space before the equals sign (`=`) now that there is an `if` condition above?